apps/trading/e2e/pyproject.toml

```diff
@@ -26,4 +26,7 @@ log_cli_date_format = "%Y-%m-%d %H:%M:%S"
 log_cli_level = "INFO"
 log_file_format = "%(asctime)s - %(name)s - %(levelname)s: %(message)s"
 log_file_date_format = "%Y-%m-%d %H:%M:%S"
-log_file_level = "INFO"
\ No newline at end of file
+log_file_level = "INFO"
+markers = [
```

Comment:
What does this do?

Reply:
It works very simmilar to the groups, annoying with pytest you can't manually kick off runs of groups, but you can use markers.
So I'm using this to run the shared vega tests only, locally.